app/Traits/UniqueQueue.php

```
@@ -39,4 +46,17 @@ public function run(string $key, Closure $callback): void
             $this->release(30);
         }
     }
+
+    protected function isTransientDatabaseError(Throwable $e): bool
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $e. Configured minimum length is 3. (undefined)

(ShortVariable)